### PR TITLE
Apply stale filter from URL

### PR DESCRIPTION
**Pull request by karelhala**

### CODE REVIEW

File: src/actions.js [hunk 1]
```diff
@@ -17,6 +17,14 @@ export const deleteEntity = (systems, displayName) => ({
     }
 });
 
+export const setFilter = (filterValue, filterKey) => ({
+    type: SET_INVENTORY_FILER,
```
Comment (rvsia) [inline src/actions.js]: Filer> filter

Comment (karelhala) [reply src/actions.js]: Nice catch!

File: src/Routes.js [hunk 2]
```diff
@@ -35,6 +36,7 @@ function checkPaths(technology, app) {
 
 export const Routes = ({ childProps: { history } }) => {
     const pathName = window.location.pathname.split('/');
+    const { status } = queryString.parse(location.search);
```
Comment (Hyperkid123) [inline src/Routes.js]: did you consider using brosers `URLSearchParams` instead of the library?

Comment (karelhala) [reply src/Routes.js]: Good point! This library looks slightly easier to use, but better to use browser build in features than custom libraries.

**Pull request merged**